neo/public/stylesheets/main.css

```diff
@@ -31,11 +34,10 @@ body {
 }
 
 time-series {
-  position: absolute;
-  top: 82%;
-  left: 15%;
-  width: 80%;
-  height: 15%;
+  z-index: 1000;
+  pointer-events: none;
+  display: inline-block;
+  position: relative;
```

Comment:
I'm not familiar with css, could you teach me what's the meaning of each values? thanks